app/models/petition.rb

```diff
@@ -100,6 +102,10 @@ class Petition < ActiveRecord::Base
   scope :with_response, -> { where.not(response_summary: nil, response: nil) }
   scope :with_debate_outcome, -> { joins(:debate_outcome) }
 
+  scope :without_debate_date, -> { where.not(id: DebateOutcome.select(:petition_id).uniq) }
```

Comment:
I think we could do this without the ``DebateOutcome.select(:petition_id).uniq`` too.  Something like:

```ruby
eager_load(:debate_outcome).where(debate_outcomes: {id: nil})
```

This does a left outer join on debate_outcomes and throws away the rows that have a joined table.  However, by using eager_load we force AR to load the debate_outcome objects when we know there won't be any, so we might want to look at using ARel (or just a string) to do the join without asking AR to load the objects.